pkg/controller/installation/products/config/monitoring.go

```diff
@@ -35,7 +39,7 @@ func (r *Monitoring) GetProductName() v1alpha1.ProductName {
 }
 
 func (r *Monitoring) GetProductVersion() v1alpha1.ProductVersion {
-	return v1alpha1.ProductVersion(r.Config["VERSION"])
+	return v1alpha1.ProductVersion(v1alpha1.OperatorVersionMonitoring)
```

Comment:
This makes sense.
@philbrookes  or @matskiv  Should all other products get the version for https://github.com/integr8ly/integreatly-operator/blob/8859df57ca42278c59821437ca5709f68b0ee707/pkg/apis/integreatly/v1alpha1/installation_types.go#L65-L78 as well?

Reply:
One and the same thing with regards the Monitoring version, but I can make it easier to read. Let me push a change there.

Reply:
We do have a series of version variables too, but prefer to find the version dynamically, so that we don't have to keep it upto date.